app/components/subcase/timeline.js

```diff
@@ -54,6 +72,9 @@ export default class SubcaseTimeline extends Component {
         phases.push({
           label: this.intl.t('activity-phase-approved-on-agenda'),
           date: meeting.plannedStart,
+          meetingId: meeting.id,
+          agendaId: agenda.id,
+          agendaitemId: firstAgendaitemOfActivity.id
         });
```

Comment:
Several scenarios came up.
Editors got links to approved agenda, even if there was a designagenda that came after.
So 2 links right above each other to different agenda versions.
Was confusing.
Changed it so the link always goes to known latest agenda (design or approved)
Viewers only have approved so that is fine.
Editors only have 1 link and it always points to the most recent agenda (design or approved)
Only side effect > we used to only show "ingediend" when not approved yet, now we also show "geagendeerd op" which seems fine because we need the link now (removal of the top link). Only editors will see this.